install_node/FreenetTray.exe.config

```diff
@@ -1,30 +0,0 @@
-<?xml version="1.0"?>
-<configuration>
-<configSections>
-    <sectionGroup name="userSettings" type="System.Configuration.UserSettingsGroup, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089">
-        <section name="FreenetTray.Properties.Settings" type="System.Configuration.ClientSettingsSection, System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" allowExeDefinition="MachineToLocalUser" requirePermission="false"/>
-    </sectionGroup>
-</configSections>
-<startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.0"/></startup><userSettings>
-        <FreenetTray.Properties.Settings>
-            <setting name="StartIcon" serializeAs="String">
-                <value>False</value>
-            </setting>
-            <setting name="StartFreenet" serializeAs="String">
-                <value>False</value>
-            </setting>
-            <setting name="UseBrowser" serializeAs="String">
-                <value>Auto</value>
-            </setting>
-            <setting name="ShowSlowOpenTip" serializeAs="String">
-                <value>True</value>
-            </setting>
-            <setting name="LogLevel" serializeAs="String">
-                <value>Off</value>
-            </setting>
-            <setting name="CustomLocation" serializeAs="String">
-                <value />
-            </setting>
-        </FreenetTray.Properties.Settings>
-    </userSettings>
-</configuration>
```

Comment:
It would be nice to accompany this with a PR to the release scripts which builds the tray binary and puts it in the right place.

Reply:
I'm going to look at the release scripts today